test/e2e/swaps/shared.js

```diff
@@ -28,19 +41,27 @@ const buildQuote = async (driver, options) => {
   await driver.wait(async () => {
     const tokens = await driver.findElements('.searchable-item-list__item');
     return tokens.length > 1;
-  }, 10000);
+  }, 20000);
```

Comment:
Why do we need to wait this long?

Reply:
I've increased it just to be on the safe side if token loading would get little longer than 10s.